docker/docker-compose.yml

```diff
@@ -453,5 +453,22 @@ services:
       aws_cli:
         condition: service_healthy
 
+  camunda-tools:
+    healthcheck:
+      test: ["CMD", "java", "/app/scripts/HealthCheck.java", "http://localhost:8080/actuator/health", "||", "exit", "1"]
```

Comment:
I still don't get why we have this Healthcheck file, alpine contains `wget`. Would be worth trying to use that?

https://pkgs.alpinelinux.org/packages?name=wget&branch=v3.13

Reply:
What I've noticed is that, the healthcheck can return say, 503 (unavailable) but we were classing that as healthy because curl process returned 0. 
We only need this for docker-compose, so may be over cooking it!